pkg/app/src/components/Forms/Campaign/form.tsx

```diff
@@ -137,13 +155,40 @@ export function Form({ organizationId, cancel, currentStep, setStep }: Component
 	const checkBackButtonState = () => {
 		return currentStep == 0
 	}
+	const handleCancelButton = useCallback(() => {
+		cancel()
+		setOpenModal(false)
+	}, [cancel])
 
 	if ([0, 1, 2].indexOf(currentStep) === -1) {
 		return null
 	}
 
 	return (
 		<>
+			<Modal
```

Comment:
create a component that can be used more often

Reply:
Will handle it in a separate ticket